src/openrct2/actions/RideEntranceExitRemoveAction.cpp

```diff
@@ -94,14 +94,15 @@ GameActions::Result RideEntranceExitRemoveAction::Query() const
     // If we are trying to remove a ghost and the element we found is real, return an error, but don't log a warning
     if (entranceElement != nullptr && (GetFlags() & GAME_COMMAND_FLAG_GHOST) && !(entranceElement->IsGhost()))
     {
-        return GameActions::Result(GameActions::Status::InvalidParameters, STR_NONE, STR_NONE);
+        return GameActions::Result(GameActions::Status::InvalidParameters, STR_ERR_GHOST_ELEMENT_NOT_FOUND, STR_NONE);
     }
     else if (entranceElement == nullptr)
     {
         LOG_WARNING(
-            "Track Element not found. x = %d, y = %d, ride = %u, station = %u", _loc.x, _loc.y, _rideIndex.ToUnderlying(),
+            "Entrance not found. x = %d, y = %d, ride = %u, station = %u", _loc.x, _loc.y, _rideIndex.ToUnderlying(),
```

Comment:
```suggestion
            "Entrance element not found. x = %d, y = %d, ride = %u, station = %u", _loc.x, _loc.y, _rideIndex.ToUnderlying(),
```

Reply:
I did see several more `LOG_WARNING` and `LOG_ERROR` messages which need to be updated, I will submit another PR when this one is finished